dev/fattest.simplicity/src/com/ibm/websphere/simplicity/ShrinkHelper.java

```diff
@@ -103,8 +103,25 @@ public static void exportDropinAppToServer(LibertyServer server, Archive<?> a) t
      * @param printArchiveContents Whether or not to log the contents of the archive being exported
      */
     public static Archive<?> exportArtifact(Archive<?> a, String dest, boolean printArchiveContents) {
+            return exportArtifact(a, dest, printArchiveContents, false);
+    }
+
+    /**
+     * Writes an Archive to a a file in the target destination
+     * with the file name returned by a.getName(), which should include the
+     * file type extension (ear, war, jar, rar, etc).
+     *
+     * @param a The archive to export as a file
+     * @param dest The target folder to export the archive to (i.e. publish/files/apps)
+     * @param printArchiveContents Whether or not to log the contents of the archive being exported
+     * @param overWrite Wheather or not to overwrite an existing artifact
+     */
+    public static Archive<?> exportArtifact(Archive<?> a, String dest, boolean printArchiveContents, boolean overWrite) {
         Log.info(c, "exportArtifact", "Exporting shrinkwrap artifact: " + a.toString() + " to " + dest);
         File outputFile = new File(dest, a.getName());
+        if (outputFile.exists() && ! overWrite) {
```

Comment:
lets add a `Log.info` here indicating that we are not writing the archive to disk because it already exists.  Will probably prevent someone from going insane debugging this situation in the future =)

Reply:
Sounds good to me